packages/ui/react-ui-data/src/components/ViewEditor/ViewEditor.tsx

```diff
@@ -102,22 +102,23 @@ export const ViewEditor = ({
       )}
 
       {!readonly && !field && (
-        <div className='flex justify-center p-2'>
-          <div className='flex gap-2'>
-            <Tooltip.Root>
-              <Tooltip.Trigger asChild>
-                <Button onClick={handleAdd} disabled={view.fields.length >= VIEW_FIELD_LIMIT}>
-                  <Icon icon='ph--plus--regular' />
-                </Button>
-              </Tooltip.Trigger>
-              <Tooltip.Portal>
-                <Tooltip.Content classNames='z-[12]'>
-                  {view.fields.length >= VIEW_FIELD_LIMIT ? t('field limit reached') : t('add field')} <Tooltip.Arrow />
-                </Tooltip.Content>
-              </Tooltip.Portal>
-            </Tooltip.Root>
-          </div>
-        </div>
+        <ButtonGroup classNames='p-2 justify-center'>
+          <Tooltip.Root>
+            <Tooltip.Trigger asChild>
+              <IconButton
```

Comment:
Why use `ButtonGroup` if there is just the one button? I can see the argument that it will scale to accommodate more buttons certainly, just curious.